run.sh

```diff
@@ -90,7 +92,7 @@ echo "this shouldn't take more than a minute.."
 num=0
 while true; do
   num=$(wget --quiet -O - "http://$graphitemon_host:8000/render/?target=graphite-watcher.$env.num_metrics&from=-2min&until=-10s&format=raw" | sed -e 's#.*,##')
-  [ "$num" == "${total}.0" ] && break
+  (( $(echo  "$num >=  $total"|bc -l) )) && break
```

Comment:
the old code was doing string comparison. I needed to change the comparison form "==" to ">=" due to the number of metrics far exceeding the expected Total.   Bash cant to numeric comparison on floats.